src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs

```diff
@@ -1792,7 +1792,7 @@ public static SelectionHandleImage GetSelectionHandleImageStruct(PropertyMap lef
         /// <param name="textEditor">The textEditor control from which the text is copied.</param>
         /// <returns>The copied text.</returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
please remove this comment `// This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)`